docs/manual/configuration/index.rst

```diff
@@ -85,6 +85,32 @@ Output::
   ---------------  -----------------  -----
   /tmp/custom.yml  backends.foo.type  mongo
 
+Keymap
+******
+
+Keymap is used to map external identifiers with internal identifiers. Storage (Backend) can configured.
+By default Sqlite is used like on configuration example above, but it can be changed to other,
+more faster and robust storages. Here full list of options:
```

Comment:
"More faster" is gramatically incorrect, due to it already being a comparative form of the word "fast", "robust" on the other hand, should have "more" before:
```suggestion
faster and more robust storages. Here full list of options:
```